src/ess/polarization/base.py

```diff
@@ -0,0 +1,200 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+from typing import NewType, TypeVar
+
+import sciline as sl
+import scipp as sc
+
+Up = NewType('Up', int)
+Down = NewType('Down', int)
+Unpolarized = NewType('Unpolarized', int)
+Spin = TypeVar('Spin', Up, Down, Unpolarized)
+PolarizerSpin = TypeVar('PolarizerSpin', Up, Down)
+AnalyzerSpin = TypeVar('AnalyzerSpin', Up, Down)
+
+Analyzer = NewType('Analyzer', str)
+Polarizer = NewType('Polarizer', str)
+Cell = TypeVar('Cell', Analyzer, Polarizer)
+
+WavelengthBins = NewType('WavelengthBins', sc.Variable)
+RawEventData = NewType('RawEventData', sc.DataArray)
+
```

Comment:
What again is "RawEventData"? Is it the measured intensities for any spin-channel?

Reply:
A placeholder for all the recorded events in a file (or combination of multiple files).